allure-generator/src/main/javascript/components/tree/TreeView.js

```diff
@@ -44,10 +39,10 @@ class TreeView extends View {
     onNodeClick(e) {
         this.$(e.currentTarget).parent().toggleClass('node__expanded');
         const uid = this.$(e.currentTarget).data('uid');
-        if (uid in  this.treeState) {
-            delete this.treeState[uid];
+        if (this.state.has(uid)) {
+            this.state.unset(uid, true);
```

Comment:
`true` is not needed.
Look at the method signature: http://backbonejs.org/#Model-unset